webpack/assets/javascripts/react_app/components/BreadcrumbBar/__tests__/__snapshots__/BreadcrumbBarReducer.test.js.snap

```diff
@@ -33,6 +35,20 @@ Object {
   "requestError": null,
   "resourceSwitcherItems": Array [],
   "resourceUrl": "some/url",
+  "searchQuery": undefined,
```

Comment:
Is that a problem? I think it's the only case `searchQuery` is `undefined` in the store.

Reply:
it's not a problem, there is no `searchQuery` in the options hash

just beneath it there's a `should handle BREADCRUMB_BAR_RESOURCES_REQUEST with search query'` test